app/controllers/users_controller.rb

```diff
@@ -0,0 +1,74 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  # GET /users
```

Comment:
Remove ALL comments in this file. They are auto-generated by scaffolding command. Of course, you don't need them 'cause you can totally understand the codes related to them. Let the codes tell your story!